fix: MetalLB documentation for air-gapped install (#4087)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/install/air-gapped/index.md

  - Management cluster must connect to the attached cluster's API server.
  - Management cluster must connect to load balancers created by some platform services.

> - A configuration file that you will adapt to your needs using the steps outlined on this page. Make sure to create that file using the following command:

Reviewer: nit for style

```suggestion
- A configuration file that you will adapt to your needs using the steps outlined in this topic. Make sure to create that file using the following command:
```

## Install on Konvoy

> 1. Adapt the configuration file for the air-gapped deployment by changing the `.apps.kommander` section. Ensure you use the actual version number everywhere `${VERSION}` appears:

Reviewer: Because this is added to the pre-reqs up above, but this is some lines after that in the topic, I think we should just add a note here:

```suggestion
1. Adapt the configuration file created from running `kommander install --init > install.yaml` for the air-gapped deployment by changing the `.apps.kommander` section. Ensure you use the actual version number everywhere `${VERSION}` appears:
```
